src/translations/es.json

```
@@ -10,192 +10,191 @@
-    "your": "su"
+    "ranked": "Competitivo",
+    "casual": "Amistoso",
+    "your": "tu"
   },
   "login": {
     "noAccount": "¿No tienes una cuenta?",
     "haveAccount": "¿Ya tienes una cuenta?",
     "title": "Bienvenido a Cuttle.Cards",
     "subtitle": "¡El juego de cartas que te enganchará desde la primera mano!",
-    "formHeaderLogin": "¡Inicia sesión para empezar!",
+    "formHeaderLogin": "¡Inicia sesión para comenzar!",
     "formHeaderSignup": "Regístrate ahora para empezar a jugar y prepárate para sumergirte en el mundo de Cuttle.",
-    "quote": "\"Cuttle es un juego rápido y agudo construido completamente sobre excelentes mecánicas. Es el tipo de juego, si lo hubiera conocido en la universidad, que habría desgastado las barajas jugando sin parar.\"",
+    "quote": "\"Cuttle es un juego ágil y rápido construido completamente sobre excelentes mecánicas. Es el tipo de juego con el que, si lo hubiera conocido en la universidad, habría desgastado las barajas jugando sin parar.\"",
     "quoteCite": "Richard Garfield - Creador de Magic: The Gathering",
-    "content": "Cuttle es un juego de cartas de batalla para 2 jugadores que se juega con un mazo estándar de 52 cartas. Tiene la sutileza estratégica de los juegos de cartas coleccionables como Magic, con el equilibrio elegante de un mazo estándar, ¡y puedes jugarlo gratis! ¡Pon a prueba tu temple en el juego de cartas más profundo bajo el mar!",
-    "content2": "Sé el primero en alcanzar 21 puntos en esta explosiva batalla de ingenio. Monta una ofensiva valiente mientras interrumpes a tu oponente con trucos malvados. ¿Tienes lo necesario para convertirte en el Señor de las Profundidades?",
+    "content": "Cuttle es un juego de batalla de cartas para 2 jugadores que se juega con una baraja de naipes estándar de 52 cartas. Tiene la sutileza estratégica de los juegos de cartas coleccionables como Magic, con el equilibrio elegante de un mazo estándar, ¡y puedes jugarlo gratis! ¡Pon a prueba tu habilidad en el juego de cartas más profundo bajo el mar!",
+    "content2": "Sé el primero en alcanzar 21 puntos en esta explosiva batalla de ingenio. Prepara una valiente ofensiva mientras frustras los planes de tu rival con malvados trucos. ¿Tienes lo necesario para convertirte en el Señor de las Profundidades?",
     "readRules": "Leer las reglas",
     "joinDiscord": "Únete a nuestra comunidad en Discord",
     "snackbar": {
-       "usernameIsTaken": "That username is already registered to another user; try logging in!",
-       "userNotFound": "Could not find that user with that username. Try signing up!",
-       "wrongPassword": "Username and password do not match"
+       "usernameIsTaken": "Ese nombre de usuario ya está registrado; prueba a iniciar sesión", 
+       "userNotFound": "No se pudo encontrar ese nombre de usuario; prueba a registrarte",
+       "wrongPassword": "El nombre de usuario y la contraseña no coinciden"
      }
   },
-  "lobby": {
+  "lobby":{
```

We generally keep a whitespace before the {
```suggestion
  "lobby": {
```